apps/admin_api/lib/admin_api/v1/controllers/token_controller.ex

```diff
@@ -94,6 +104,28 @@ defmodule AdminAPI.V1.TokenController do
     end
   end
 
+  defp do_create(conn, %{"amount" => _, "blockchain_address" => _}) do
+    handle_error(
+      conn,
+      :invalid_parameter,
+      "Can't specify both `blockchain_address` and `amount`."
+    )
+  end
+
+  defp do_create(conn, %{"blockchain_address" => blockchain_address} = attrs) do
```

Comment:
`/token.set_blockchain_address` requires a specific `:set_blockchain_address` permission, but this one allows it with a less strict the token's `:create` permission.

We could either:
1. Separate the existing token's `:create` permission into `:create_internal` and `:create_blockchain`
2. Just add `authorize(:set_blockchain_address, conn.assigns, token)` to this function

I kinda like the 2nd way, more composable.